### ISSUE TITLE
The serif-font has disappeared

### ISSUE THREAD

**Issue by ferpaks** [whyt-byte/Blue-Topaz_Obsidian-css#506]
**TLDR:** After some January update old serif-font has disappeared.
It worked before January updates and worked in (_Obsidian v0.15.7。uid [20220722]_) and (_20221017_) version of CSS file. I has no backups of newer versions.

I loved the **Bookerly** font, it looked nice. But after January update it has disappeared yet again. And the default font is used now. Creating of new vault didn't fixed the problem.

In the previous time when the same thing happened, after some weeks (in next theme update) serif font has returned. And I was happy.

Could you please return this font again? The default font is ugly for Cyrillic script. If you think the problem is caused by my location, I'm in **Russia**. But the font is still **Bookerly** in other Vaults, which use older versions of CSS files.

![Pasted image 20230125071339](https://user-images.githubusercontent.com/92272883/214465460-51f664ec-e355-4df6-9362-4821d7b175a3.png)

**Comment by cumany** [comment]
Because some people complain about the size of the theme is too big, so the font file is put in a separate css snippet, you can download[ this snippet](https://github.com/cumany/Blue-topaz-examples/blob/main/.obsidian/snippets/%E3%80%90FONT%E3%80%91Consolas-Bookerly.css) and put it in the snippet folder, just enable it in `obsidian-appearance-snippet`.

**Comment by ferpaks** [comment]
Yeah, that helped. Thank you!
